_docs-sources/foundations/landing-zone/add-aws-account.md

```diff
@@ -6,46 +6,50 @@ This document provides instructions for provisioning a new AWS account using Gru
+
+  :::info
+
+  Your GitHub org MUST have fine-grained personal access tokens enabled before you can create the access tokens and secrets!
+
+  :::
+
+  :::info
+
+  Instructions for configuring access tokens and secrets can be found in the respective repo Readme files, as well as in [the Machine Users](/pipelines/security/machine-users) guide.
+
+  :::
 
 ## 1. Create an AWS account request file
 
-To initiate the process, create an `account-<AWS-ACCOUNT-NAME>.yml` file in the `_new-account-requests` folder located in the root of your `infrastructure-live` repository. This file will be used to create a pull request and add the new account to your organization. The file should have the following format:
-
-```yaml account-<AWS-ACCOUNT-NAME>.yml
-account_name: <AWS-ACCOUNT_NAME>
-account_email: <ROOT-EMAIL-FOR-ACCOUNT>
-organizational_unit_name: <OU-TO-ADD-ACCOUNT-TO>
-aws_region: <REGION-T0-DEPLOY-BASELINES>
-org_name_prefix: <ORG-NAME-PREFIX-FOR-RESOURCES>
-sso_user_first_name: <SSO-USER-FIRST-NAME>
-sso_user_last_name: <SSO-USER-LAST-NAME>
-sso_user_email: <SSO-USER-EMAIL>
-account_baseline_modules_version: <ACCOUNT-BASELINE-MODULES-VERSION>
-requested_by: <GITHUB_USER_ID_OR_EMAIL>
-```
-
-- `account_name`: The name of the account in AWS.
-- `account_email`: The email address to associate with the account.
-- `organizational_unit_name`: The name of the Organizational Unit(OU) where the new account will be added. Ensure this exactly matches an existing OU name in your organization, for example, “Security”.
-- `aws_region`: The AWS region where the Gruntwork account baselines will be deployed. e.g., `us-east-1`.
-- `org_name_prefix`: The prefix to use for some resources created by Gruntwork modules. For instance, S3 bucket files will be prefixed with this value
-- `sso_user_first_name`: The first name of the user to create in AWS IAM Identity Center (Successor to AWS Single Sign-On(SSO)).
-- `sso_user_last_name`: The last name of the user to create in AWS IAM Identity Center.
-- `sso_user_email`: The email address of the user to create in AWS IAM Identity Center. This user will be able to login to the new account using AWS IAM Identity Center.
-- `account_baseline_modules_version`: The version of the your account baseline modules to use for the new account. e.g., `v0.0.1`
-- `requested_by`: The GitHub user ID or email address of the entity requesting the new account, for audit purposes.
-
-:::note
-
-If you have configured GitHub Actions in your `infrastructure-live` repository with a Gruntwork Landing Zone Account Factory workflow, you can invoke that workflow via the GitHub UI or programmatically. This workflow will automatically create the account request file and open a pull request on your behalf. After doing so, jump to [step 3](#3-review-and-merge-the-account-request-pr).
+To initiate the process, you will use the GitHub Actions workflow in your `infrastructure-live` repo.
```

Comment:
That's a great point, I don't think we want to remove it entirely. 

I've added the old docs back. I put two drop down tabs, one for the GitHub Action Workflow instructions, which I set to auto-open when the page loads, as well as the original manual Account Request File workflow which will be closed when the page loads.

Reply:
When we present two alternative approaches like this, I think it would be better to leverage [the built-in tabs feature](https://docusaurus.io/docs/markdown-features/tabs). Everything else about this change looks good.